src/Microsoft.Extensions.Configuration.AzureAppConfiguration/AzureAppConfigurationProvider.cs

```diff
@@ -23,15 +23,17 @@ internal class AzureAppConfigurationProvider : ConfigurationProvider, IConfigura
         private bool _optional;
         private bool _isInitialLoadComplete = false;
         private readonly bool _requestTracingEnabled;
-
-        private readonly ConfigurationClient _client;
+        private readonly IConfigurationClientProvider _configClientProvider;
         private AzureAppConfigurationOptions _options;
         private Dictionary<string, ConfigurationSetting> _applicationSettings;
         private Dictionary<KeyValueIdentifier, ConfigurationSetting> _watchedSettings = new Dictionary<KeyValueIdentifier, ConfigurationSetting>();
         private RequestTracingOptions _requestTracingOptions;
 
         private readonly TimeSpan MinCacheExpirationInterval;
 
+        // This constant is necessary because HttpStatusCode.TooManyRequests is only available in netstandard2.1 and higher.
+        private const int HttpStatusCodeRequestThrottled = 429;

```

Comment:
If we need this, it's worth it's own static class for namespacing. like `HttpStatusCodes`.

Reply:
Moved to a separate type `HttpStatusCodes`.